hongyujin/week02/Baekjoon_10984.py

```diff
@@ -0,0 +1,14 @@
+t = int(input())  # 학기의 수 입력
+
+for _ in range(t):  # for문으로 t 반복문
+    n = int(input())  # 과목의 수
+    credit = 0
+    grade = 0
+
+for _ in range(n):
+    c, g = map(float, input().split())
+    credit += int(c)
+    grade += float(c) * float(g)  # 성적은 float로 바꿔서 계산
```

Comment:
```suggestion
    grade += c * g 
```
이미 9번째 줄에서 float형으로 형변환을 마쳐서 이런식으로 적어도 괜찮을거 같습니다..!!